<?xml version="1.0" encoding="UTF-8"?>
<!-- SPDX-FileCopyrightText: Copyright (c) Kitware Inc. -->
<!-- SPDX-License-Identifier: BSD-3-Clause -->
<ui version="4.0">
 <class>pqConfigureCategoriesDialog</class>
 <widget class="QDialog" name="pqConfigureCategoriesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Categories</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0" rowspan="4">
    <widget class="QGroupBox" name="filtersGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Available Filters</string>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="pqSearchBox" name="searchBox" native="true">
        <property name="settingKey" stdset="0">
         <string notr="true">showAdvancedProperties</string>
        </property>
        <property name="advancedSearchEnabled" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTreeWidget" name="defaultCategoriesTree">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DragOnly</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::CopyAction</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4">
    <widget class="QGroupBox" name="customGroup">
     <property name="title">
      <string>Current Categories</string>
     </property>
     <layout class="QGridLayout">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QToolButton" name="addFilter">
        <property name="toolTip">
         <string>Add Filter to custom menu</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/pqWidgets/Icons/pqNavForward.svg</normaloff>:/pqWidgets/Icons/pqNavForward.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="setIcon">
        <property name="toolTip">
         <string>Set an icon for the current proxy</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/pqWidgets/Icons/pqChooseIcon.svg</normaloff>:/pqWidgets/Icons/pqChooseIcon.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="addCategory">
        <property name="toolTip">
         <string>Create a category</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/QtWidgets/Icons/pqPlus.svg</normaloff>:/QtWidgets/Icons/pqPlus.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="addSubCategory">
        <property name="toolTip">
         <string>Insert a subcategory</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/pqWidgets/Icons/pqSubElement.svg</normaloff>:/pqWidgets/Icons/pqSubElement.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="useAsToolbar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Toolbar</string>
        </property>
        <property name="toolTip">
          <string>Add a toolbar for this category</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="6">
       <widget class="QToolButton" name="remove">
        <property name="toolTip">
         <string>Delete selected entry</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/QtWidgets/Icons/pqMinus.svg</normaloff>:/QtWidgets/Icons/pqMinus.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QToolButton" name="resetAll">
        <property name="toolTip">
         <string>Reset categories</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../Widgets/Resources/pqComponents.qrc">
          <normaloff>:/pqWidgets/Icons/pqReset.svg</normaloff>:/pqWidgets/Icons/pqReset.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="3" colspan="8">
       <widget class="pqFavoritesTreeWidget" name="customCategoriesTree">
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DragDrop</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::MoveAction</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>true</bool>
        </property>
        <attribute name="headerStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqSearchBox</class>
   <extends>QWidget</extends>
   <header>pqSearchBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqFavoritesTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>pqFavoritesTreeWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../pqComponents.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>pqConfigureCategoriesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>pqConfigureCategoriesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
